test/array-includes.ts

```diff
@@ -0,0 +1,26 @@
+import test from 'ava';
+import {arrayIncludes} from '../source/index.js';
+
+test('arrayIncludes()', t => {
+	const values = ['a', 'b', 'c'] as const;
```

Comment:
Can you also do an assertions that confirms that it preserves the readonly-ness of an array?

Reply:
I'm not sure how this would affect the `readonly` status of the array. The type-guard only applies to the second argument. 

Would you want something like this?

```ts
// @ts-expect-error
values.push
```

Reply:
I just commented on an arbitrary line. The content above is not related to my comment.

Reply:
> Would you want something like this?

Yes

Reply:
I added something which, I hope, is not too ugly.